tool/tsh/tsh.go

```diff
@@ -1583,7 +1583,8 @@ func onLogin(cf *CLIConf) error {
 	}
 
 	// get any "on login" alerts
-	alerts, err := tc.GetClusterAlerts(cf.Context, types.GetClusterAlertsRequest{
+	alertCtx, _ := context.WithTimeout(cf.Context, constants.TimeoutGetClusterAlerts)
```

Comment:
Different pkg but same code as in `tctl/common` maybe this should be abstracted out and shared.